analysis/load_data_example.py

```diff
@@ -0,0 +1,90 @@
+
+# coding: utf-8
+
+# ## Loading Data...
+# This notebook is an example showing how to load the data with a custom loading class
+
+# In[5]:
+
+# Add repo directory to path so that we can do the relative import below
+# Probably a better way to do this!
+import sys
+sys.path.append('/home/gus/Repos/kaggle')
```

Comment:
yea, may not want to hard code paths from your local machine. to make it a bit more independent you can add folders to the python path "outside" the python process, i.e. before launching, with something like

```
$ export PYTHONPATH=<path>
$ jupyter-notebook
```

or add a minimal `setup.py` (see our other projects) and install the local project with `pip install -e`